Resources/SteamLanguage/emsg.steamd

```diff
@@ -759,36 +781,36 @@ enum EMsg
 	OGSWriteAppSessionRow = 3406;
 
 	BaseBRP = 3600;
-	BRPStartShippingJobs = 3601;
-	BRPProcessUSBankReports = 3602;
-	BRPProcessGCReports = 3603;
-	BRPProcessPPReports = 3604;
-	BRPSettleNOVA = 3605; removed
-	BRPSettleCB = 3606; removed
-	BRPCommitGC = 3607;
-	BRPCommitGCResponse = 3608;
-	BRPFindHungTransactions = 3609;
-	BRPCheckFinanceCloseOutDate = 3610;
-	BRPProcessLicenses = 3611;
-	BRPProcessLicensesResponse = 3612;
-	BRPRemoveExpiredPaymentData = 3613;
-	BRPRemoveExpiredPaymentDataResponse = 3614;
-	BRPConvertToCurrentKeys = 3615;
-	BRPConvertToCurrentKeysResponse = 3616;
-	BRPPruneCardUsageStats = 3617;
-	BRPPruneCardUsageStatsResponse = 3618;
-	BRPCheckActivationCodes = 3619;
-	BRPCheckActivationCodesResponse = 3620;
-	BRPCommitWP = 3621;
-	BRPCommitWPResponse = 3622;
-	BRPProcessWPReports = 3623;
-	BRPProcessPaymentRules = 3624;
-	BRPProcessPartnerPayments = 3625;
-	BRPCheckSettlementReports = 3626;
-	BRPPostTaxToAvalara = 3628;
+	BRPStartShippingJobs = 3601; removed
+	BRPProcessUSBankReports = 3602; removed
+	BRPProcessGCReports = 3603; removed
+	BRPProcessPPReports = 3604; removed
+	BRPSettleNOVA = 3605; removed removed

```

Comment:
`removed removed`

Guessing we're lucky steamlanguage doesn't mind this.

Reply:
perhaps it should?